jjb/duffy/testing/sig-paas-testing.yml

```diff
@@ -0,0 +1,91 @@
+- defaults:
```

Comment:
Not yet. I want to keep the duffy testing stuff there for now. Specifically, I plan to move it over to a simple job to test spinning up and down a cluster. Eventually, I want to have it test linch-pin.